From 64e11d462a73b70f58cfeeaf4436249ca192d032 Mon Sep 17 00:00:00 2001
Message-Id: <64e11d462a73b70f58cfeeaf4436249ca192d032.1683758992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 10 May 2023 23:47:34 +0100
Subject: [PATCH] gnu: python-pytest-httpx: Update to 0.22.0, fix build.

* gnu/packages/python-check.scm (python-pytest-httpx): Update to 0.22.0,
fix build.
[build-system]: Use pyproject-build-system.
[arguments]: Remove as no longer required.
---
 gnu/packages/python-check.scm | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/gnu/packages/python-check.scm b/gnu/packages/python-check.scm
index f272a252fc..6c5f45d4b5 100644
--- a/gnu/packages/python-check.scm
+++ b/gnu/packages/python-check.scm
@@ -2530,7 +2530,7 @@ (define-public python-pytest-parawtf
 (define-public python-pytest-httpx
   (package
     (name "python-pytest-httpx")
-    (version "0.21.0")
+    (version "0.22.0")
     (source
      (origin
        ;; pypi package doesn't include the tests
@@ -2540,16 +2540,8 @@ (define-public python-pytest-httpx
              (commit (string-append "v" version))))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "12mcy1f5d5cq3rqrqgi2ar0qvzw62ibys17hw6dsdfd0j2syck4r"))))
-    (build-system python-build-system)
-    (arguments
-     `(#:phases
-       (modify-phases %standard-phases
-         (replace 'check
-           (lambda* (#:key tests? #:allow-other-keys)
-             (when tests?
-               (setenv "PYTHONPATH" (getcwd))
-               (invoke "pytest" "-vv")))))))
+        (base32 "1ncpd74hmsz4sadvjg99fnfscxpgh3mc2siini0dhxzwgwdkk5i7"))))
+    (build-system pyproject-build-system)
     (propagated-inputs (list python-httpx))
     (native-inputs (list python-pytest python-pytest-asyncio))
     (home-page "https://colin-b.github.io/pytest_httpx/")

base-commit: b4e5844700b2304bfde451322feb5797bf0c6179
-- 
2.39.2

